packages/components/src/Disclosure/Disclosure.tsx

```diff
@@ -16,8 +16,9 @@ interface DisclosureProps {
 
   /**
    * Title for the disclosure pane.
+   * If ReactElement[] is provided, it must be wrapped in a container element (not a Fragment).
    */
-  readonly title: string | ReactNode | ReactNode[];
+  readonly title: string | ReactElement | ReactElement[];
```

Comment:
nice. the type is a bit clearer that string is acceptable now in the docs

though the far right interpretation of the types is horribly unreadable. that's not because of this PR. our tooling is being a bit more verbose than is helpful.

Reply:
<img width="1490" alt="image" src="https://github.com/user-attachments/assets/6da39476-ddd1-422e-b0df-7d87c1ca14ca">

Reply:
Yep, I tried providing more specificity there, but it wasn't getting any shorter...